packages/CodeQuality/tests/Rector/Switch_/SimplifyBinarySwitchRector/Correct/correct.php.inc

```diff
@@ -0,0 +1,28 @@
+<?php
+
+if ($foo == 'my string') {
+    $result = 'ok';
+} else {
+    $result = 'not ok';
+}
+
+if ($foo == 'my string') {
+    $result = 'ok';
+}
+
+if ($foo == 'my string') {
+    $result = 'ok';
+} elseif ($foo == 'your string') {
```

Comment:
A switch to a else/if is totally no go :-1: 

My idea was just a switch with **1** case and **default**.

Reply:
Why?

Reply:
Anyhow, we shall make it at least configurable via ctor options.

To enable/disalbe 2nd case